task-impls/src/consensus.rs

```diff
@@ -486,31 +753,41 @@ where
                                     .await;
                             }
                         });
-                        if let GeneralConsensusMessage::Vote(vote) = message {
```

Comment:
This was in here for sending no votes if the proposal was incorrect/invalid.  The message is created above only if the vote is a no

Reply:
Yes, you are right!  I'll add it back in